config/zones.json

```diff
@@ -572,8 +572,8 @@
       [-49.37304648399993, 4.941122259000181]
     ],
     "capacity": {
-      "hydro": 22251,
-      "solar": 5,
+      "hydro": 22273,
+      "solar": 3,
```

Comment:
Huh interesting, do you know why solar capacity was reduced?
(Just curious.)

Reply:
I have no idea, I'm sorry...

Reply:
No worries!